pkg/controller/config_watcher.go

```diff
@@ -47,19 +47,21 @@ func (w *ConfigWatcher) Do(ctx context.Context, cancelFunc context.CancelFunc) (
 	ctx, cancelFn := context.WithCancel(ctx)
 	defer cancelFn()
 
+	log := w.log.WithField("configs", w.configPaths)
+
 	errGroup, _ := errgroup.WithContext(ctx)
 	errGroup.Go(func() error {
 		for {
 			select {
 			case <-ctx.Done():
-				w.log.Info("Shutdown requested. Finishing...")
+				log.Info("Shutdown requested. Finishing...")
 				return nil
 			case _, ok := <-watcher.Events:
 				if !ok {
 					return fmt.Errorf("unexpected file watch end")
 				}
 
-				w.log.Infof("Config file %q is updated. Sending last message before exit...", configFile)
+				log.Infof("Config updated. Sending last message before exit...")
```

Comment:
Do we know which file was updated? I think it would be handy to print it, if that's easy to do. I believe the watch event would contain this info?